src/containers/SubmitSteps/Questionary/Step1e/Step1e.tsx

```diff
@@ -0,0 +1,176 @@
+  });
+  const {
+    setDoGoBack, setTitle, setSubtitle, setType,
+  } = useHeaderContext();
+  const history = useHistory();
+  const { t } = useTranslation();
+  const { state, action } = useStateMachine(updateAction(storeKey));
+  const patientId = getPatientId();
+  const country = getCountry();
+
+  // States
+  const [activeStep, setActiveStep] = React.useState(true);
+
+  // Form
+  const {
+    control, handleSubmit, formState,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: state?.[storeKey],
+    context: {
+      country,
+    },
+    resolver: yupResolver(schemaWithoutPatient),
+  });
+  const { errors, isValid } = formState;
+
+  // Callbacks
+  const handleDoBack = React.useCallback(() => {
+    setActiveStep(false);
+    if (previousStep) {
+      history.push(previousStep);
+    } else {
+      history.goBack();
+    }
+  }, [history, previousStep]);
+
+  // Effects
+  useEffect(() => {
+    scrollToTop();
+    if (patientId) {
+      setTitle('');
+    } else {
+      setTitle(t('questionary:covidTimesTitle'));
+    }
+    if (patientId) {
+      setType('tertiary');
+    } else {
+      setType('primary');
+    }
+    setSubtitle('');
+    setDoGoBack(() => handleDoBack);
+  }, [handleDoBack, setDoGoBack, setTitle, setType, setSubtitle, patientId, t]);
+
+  // Handlers
+  const onSubmit = async (values: Step1eType) => {
+    if (values) {
+      action(values);
+      if (nextStep) {
+        setActiveStep(false);
+        history.push(nextStep);
+      }
+    }
+  };
+
+  return (
+    <MainContainer>
+      <ProgressIndicator
+        currentStep={metadata?.current}
+        totalSteps={metadata?.total}
+        progressBar
+      />
+      <QuestionText extraSpace first>
+        {t('questionary:covidTimes')}
+      </QuestionText>
+      <Controller
+        control={control}
+        name="covidTimes"
+        defaultValue=""
+        render={({ onChange, value, name }) => (
+          <QuestionInput
+            name={name}
+            value={value}
+            onChange={onChange}
+            type="number"
+            placeholder={t('questionary:covidTimesPlaceholder')}
+            autoComplete="Off"
+          />
+        )}
+      />
+
+      <QuestionText extraSpace>
+        {t('questionary:lastTimeCovidMonths')}
+        <QuestionAllApply>{t('questionary:lastTimeCovidMonthsCaption')}</QuestionAllApply>
+      </QuestionText>
+      <Controller
+        control={control}
+        name="lastTimeCovidMonths"
+        defaultValue=""
+        render={({ onChange, value, name }) => (
+          <QuestionInput
+            name={name}
+            value={value}
+            onChange={onChange}
+            type="number"
+            placeholder={t('questionary:lastTimeCovidMonthsPlaceholder')}
+            autoComplete="Off"
+          />
+        )}
+      />
+
+      {/* Bottom Buttons */}
+      <p><ErrorMessage errors={errors} name="name" /></p>
```

Comment:
Same as previous. Also, there is no "name" field here. Which error you will shown?